ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.modal.template.html

```diff
@@ -122,7 +122,7 @@ <h3>You're all set!</h3>
             <p>What would you like to do next?</p>
             <button class="btn btn-default btn-block" ng-click="$dismiss('Continue editing')">Continue to edit this blueprint</button>
             <button class="btn btn-info btn-block" ng-click="$close(REASONS.new)">Create a new blueprint</button>
-            <a class="btn btn-primary btn-block" ng-href="/brooklyn-ui-catalog/#!/bundles/catalog-bom-{{catalogBundleId}}/{{config.version}}/types/{{catalogBundleBase}}/{{config.version}}">View in catalog</a>
+            <a class="btn btn-primary btn-block" ng-href="{{getCatalogURL()}}">View in catalog</a>
```

Comment:
No need to how this button if URL is not available or not valid, consider `ng-if` to hide it until it URL is valid, or at least disable.

Reply:
@algairim  I cannot call a function as argument in an `ng-if`. I gave up the `ng-if`  because I noticed the modal title doesn't do it either. Sample: 
` <h3 class="modal-title">{{getTitle()}}</h3>`